srv/exo/bin/backup_plateforme.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash -eu
+
+# #############################################################################
+# Initialize
+# #############################################################################                                              
+SCRIPT_NAME="${0##*/}"
+SCRIPT_DIR="$( cd -P "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+SCRIPT_START_TIME=$(date +%s)
+
+# Load env settings
+source ${SCRIPT_DIR}/_setenv.sh
+# Load common functions
+source ${SCRIPT_DIR}/_functions.sh
+
+DOWNTIME_START_TIME=$(date +%s)
+# Stop it
+ssh -i ${SCRIPT_DIR}/id_rsa ${EXO_USER}@${EXO_PLF_SERVER} ${SCRIPT_DIR}/_stopPLF.sh
```

Comment:
We should rely on the system to ensure the current user can connect on the distant server without specifying the ssh key